StitchCore-iOS/StitchCore-iOS/Core/Stitch.swift

```diff
@@ -33,67 +39,26 @@ public class Stitch {
     public static func initialize() throws {
         try! StitchCore.sync(self) {
```

Comment:
[aside] @jsflax do we have a ticket for potentially eliminating these `try!`s?

Reply:
https://jira.mongodb.org/browse/STITCH-1356 I'm going to get on it once this PR is merged in.